admin/src/components/Layout/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Header from "../Header";
+import { Container } from "react-bootstrap";
+import { NavLink } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { logOut } from "../../actions";
+
+import "./style.scss";
+
+export default function Layout(props) {
+  const auth = useSelector((state) => state.auth);
+  const dispatch = useDispatch();
+  return (
+    <>
+      <Header />
+      {props.sidebar ? (
+        <Container>
+          <div className='layout-container'>
+            <nav className='sidebar'>
+              <div className='sidebar-container'>
+                <h1 className='brand'>
+                  Travling<span>Dashboard</span>
+                </h1>
+                <ul>
+                  <li>
+                    <NavLink to={"/"}>Home</NavLink>
```

Comment:
```suggestion
                    <NavLink to="/">Home</NavLink>
```
Phân biệt giúp em NavLink với Link

Reply:
Navlink có attribute active khi url với với url của page